docs/frontend.md

```diff
@@ -4,28 +4,30 @@ Ideally, you will be using a theme that employs SASS/SCSS, a styleguide, and oth
 
 Like Composer dependencies, your frontend dependencies and compiled frontend assets should not be directly committed to the project repository. Instead, they should be built during the creation of a production-ready artifact.
 
-BLT does not directly manage any of your front end dependencies or assets, but it does create opportunities for you to hook into the build process with your own custom frontend commands. Additionally, BLT ships with [Cog](https://github.com/acquia-pso/cog), a base theme that provides front end dependencies and front end build tasks compatible with BLT.
+BLT does not directly manage any of your frontend dependencies or assets, but it does create opportunities for you to hook into the build process with your own custom frontend commands. Additionally, BLT ships with [Cog](https://github.com/acquia-pso/cog), a base theme that provides front end dependencies and front end build tasks compatible with BLT.
 
 ##  Available target hooks
 
 Out-of-the-box, BLT provides an opportunity for your frontend commands to run at three different stages of the build process.
 
 You must let BLT know which commands should be run in which directories. You can do this by specifying values in `blt/blt.yml` file under the `command-hooks` key.
 
-These commands will run inside of the virtual machine, if available. This obviates the need to install frontend dependencies on the host machine.
+These commands will run inside of the virtual machine, if available. This obviates the need to install frontend dependencies on the host machine. If you want to use a node version manager such as `nvm` you should configure that inside the virtual environment's configuration file.
 
 The three following target hooks are available for frontend commands: setup, build, test.
 
 ### Setup
 
+BLT automatically setups up a site with the lightning profile and cog theme. To create a subtheme from cog see [Cog's documentation](https://github.com/acquia-pso/cog/blob/8.x-1.x/STARTERKIT/README.md#create-cog-sub-theme) for initial installation.
```

Comment:
We document fairly clearly that lightning can be changed, and if COG is setup by lightning, we should just clarify that here. By default, BLT will setup the lightning profile, which in turn will install cog as the base theme. You can use one without the other, but if you choose to use a different profile, you will need to manually configure cog.